tests/examples/voting/test_ballot.py

```diff
@@ -1,11 +1,14 @@
 import pytest
 
+PROPOSAL_1_NAME = b"Clinton" + b"\x00" * 25
```

Comment:
`web` raises the following error if they are not padded - I believe they are treated as `bytes` instead of `bytes32`.

```
E           TypeError: One or more arguments could not be encoded to the necessary ABI type. Expected types are: bytes32[2]
```